catalog/core/catalog-core-commands/src/test/java/org/codice/ddf/commands/catalog/DumpCommandTest.java

```diff
@@ -63,7 +80,7 @@
   @Test
   public void testNonExistentOutputDirectory() throws Exception {
     // given
-    DumpCommand dumpCommand = new DumpCommand();
+    DumpCommand dumpCommand = new DumpCommand(signer);
```

Comment:
Why not create a DumpCommand object that's global?

Reply:
I'm not sure why the existing unit tests function this way however if we wanted to update them we should create a new improvement issue.